Cargo.toml

```diff
@@ -22,8 +22,10 @@ bitcoin = { version = "0.32", features = ["serde", "std"], default-features = fa
 hex = { version = "0.2", package = "hex-conservative" }
 log = "^0.4"
 minreq = { version = "2.11.0", features = ["json-using-serde"], optional = true }
-reqwest = { version = "0.11",  features = ["json"], default-features = false, optional = true }
-async-std = { version = "1.13.0", optional = true }
+reqwest = { version = "0.12",  features = ["json"], default-features = false, optional = true }
```

Comment:
I think the `reqwest` version bumping should be in its own commit.

Reply:
fixed